students/jesse_miller/session04/trigrams.py

```diff
@@ -16,55 +13,62 @@ def parse_file():
     dictionary.
     Thanks for the help Luis!
     """
-    temp_dict = {}
-    sherlock_dict = {}
-    translator = str.maketrans('', '', string.punctuation)
-    sherlock_input = set(line.strip() for line in open('sherlock_small.txt'))
+    startline = 'Produced by an anonymous Project Gutenberg'\
+                'volunteer and Jose Menendez'
+    endline = 'End of the Project Gutenberg EBook'
```

Comment:
Don't worry about _startline_ and _endline_ right now, just process the entire text.